### PR TITLE
feat: add example for gdrive

### PR DESCRIPTION
**Pull request by junaire**
ref: [go-service-gdrive/issues/6](https://github.com/beyondstorage/go-service-gdrive/issues/6)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/beyondstorage/go-service-cos/v2 v2.2.0
 	github.com/beyondstorage/go-service-fs/v3 v3.4.0
 	github.com/beyondstorage/go-service-ftp v0.1.0
+	github.com/beyondstorage/go-service-gdrive v0.0.0-20210813025406-1a9502b99c89
```
Comment (JinnyYi) [inline go.mod]: How about release a `v0.1.0` for `go-service-gdrive`? We need to make a release before we adding them into docs.

Comment (Xuanwo) [reply go.mod]: Yep, we will release v0.1.0 after go-service-gdrive passed integration tests.

**Comment by abyss-w** [comment]
Maybe we should add the description of gdrive to the README.

**Comment by junaire** [comment]
> Maybe we should add the description of gdrive to the README.

Sure, I will update README after we pass the integration tests.

**Comment by Xuanwo** [comment]
Gdrive passed the integration tests, let's update gdrive's README and bump a version for it!

File: go.mod (hunk 1 above)
Comment (Xuanwo) [reply go.mod]: v0.1.0 has been released, let's update the version here.

**Comment by Xuanwo** [comment]
Please also resolve the conflicts.

**Comment by junaire** [comment]
> Please also resolve the conflicts.

Done

File: go.mod [hunk 2]
```diff
@@ -7,11 +7,12 @@ require (
...
+	github.com/beyondstorage/go-service-gdrive v0.1.0
 	github.com/beyondstorage/go-service-hdfs v0.2.1
 	github.com/beyondstorage/go-service-ipfs v0.2.0
 	github.com/beyondstorage/go-service-minio v0.2.0
 	github.com/beyondstorage/go-service-obs v1.0.0
 	github.com/beyondstorage/go-service-s3/v2 v2.4.0
 	github.com/beyondstorage/go-storage/v4 v4.7.0
-	github.com/golang/protobuf v1.5.2 // indirect
+
 )
```
Comment (JinnyYi) [inline go.mod]:
```suggestion
	github.com/beyondstorage/go-storage/v4 v4.7.0
)
```

File: new_gdrive.go [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		// work_dir: https://beyondstorage.io/docs/go-storage/pairs/work_dir
+		//
+		// Relative operations will be based on this WorkDir.
+		pairs.WithWorkDir(os.Getenv("STORAGE_GDRIVE_WORKDIR")),
+	)
+}
+
+func NewGdriveFromString() (types.Storager, error) {
+	str := fmt.Sprintf(
+		"gdrive://%s/%s?credential=%s",
```
Comment (JinnyYi) [inline new_gdrive.go]:
```suggestion
		"gdrive://%s%s?credential=%s",
```

File: go.mod (hunk 2 above)
Comment (junaire) [reply go.mod]: Hi, What's this?

Comment (JinnyYi) [reply go.mod]: How about remove the blank line?

**Pull request merged**